claim_ai/evaluation/converters/patient.py

```diff
@@ -46,6 +54,6 @@ def _get_contained_patient_location_code(self, contained_patient):
         return location_extension['valueReference']['identifier']['value']
 
     def _get_contained_patient_group(self, contained_patient):
-        # TODO: This extension is not yet available in fhir api, returns None as default
-        return next((extension['value'] for extension in contained_patient['extension']
-                     if extension['url'].endswith('povertyStatus')) or [], None)
+        group = next(extension for extension in contained_patient['extension']
+                                  if extension['url'].endswith('group'))
+        return group['valueReference']['identifier']['value']
```

Comment:
I mostly saw the references used as "Group/UUID" as short reference. Can we check if reference.reference exists take the UUID from there else search in identifier?

Reply:
Changed for group and also for location